product_local_oversea_stock_info/models/stock_quant.py

```diff
@@ -27,10 +27,14 @@ def _update_prod_tmpl_reserved_qty(self):
             rsvd_qty = 0.0
             for prod in prod_tmpl.product_variant_ids:
                 quants = self.search(
-                    [("product_id", "=", prod.id), ("usage", "=", "internal")]
+                    [
+                        ("product_id", "=", prod.id),
+                        ("sale_order_id", "!=", False),
+                        ("usage", "=", "internal"),
+                    ]
                 )
                 for q in quants:
-                    rsvd_qty += q.reserved_quantity
+                    rsvd_qty += q.quantity
```

Comment:
Another way:
```.py
                if quants:
                    rsvd_qty += sum(quants.mapped("quantity"))
```
Not sure if the performance is any better with this way, though.